deploy.sh

```diff
@@ -23,6 +23,13 @@ elif [ "$1" == "build" ]; then
 		exit 1
 	fi
 	echo "build finished"
+elif [ "$1" == "test" ]; then
+	./deploy.sh build
+	rm -rf coverage
```

Comment:
Suggest cleaning all directories, including coverage, before beginning the build process. May help prevent confusion with people looking at outdated codecov reports not realizing their build failed.

Reply:
Optionally modify `./deploy.sh clean` to clean coverage

Reply:
I ended up going with the ladder